src/main/java/frc/robot/subsystems/LiftRotater.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import frc.robot.RobotMap;
+import frc.robot.Constants;
+
+public class LiftRotater extends SubsystemBase{
+    private CANSparkMax robotLiftRotateMotorLeft = new CANSparkMax(RobotMap.robotLiftRotaterLeftMotorPort, MotorType.kBrushless);
+    private CANSparkMax robotLiftRotateMotorRight = new CANSparkMax(RobotMap.robotLiftRotaterRightMotorPort, MotorType.kBrushless);
+    private double targetVoltage = Constants.liftRotaterTargetVoltage;
+    private RelativeEncoder robotLiftRotaterLeftEncoder = robotLiftRotateMotorLeft.getEncoder();
+    public LiftRotater(){
+     // robotLiftMotor.enableVoltageCompensation(true);
+     // robotLiftMotor.setVoltage(targetVoltage);
+    }
+    /**
+     * robotLiftMotorLeft creates variable for the left lift motor.
+     * stop() stops the motor.
+     */
+  
+
+    public void stop() {
+
+        robotLiftRotateMotorLeft.stopMotor();
+        robotLiftRotateMotorLeft.set(0.0);
```

Comment:
Correct me if I'm wrong, but set(0) and stopMotor() seem like the same operation

Reply:
Yes thats correct